src/renderer/components/AircraftSection/index.tsx

```diff
@@ -470,6 +472,16 @@ const index: React.FC<TransferredProps> = (props: AircraftSectionProps) => {
         return state.liveries.map((entry) => entry.livery);
     });
 
+    function startMSFS() {
```

Comment:
**style:** to follow consistent style, make this an arrow function and call it `handleStartMsfs`

Reply:
commit 
6b1d93c